### ISSUE TITLE
how to make myself pose trajectory file

### ISSUE THREAD

**Issue by warmkid** [tinghuiz/SfMLearner#41]
now I have complete the training process, and want to output the pose trajectory according to the training model. In your page, I need to run **python test_kitti_pose.py --test_seq [sequence_id] --dataset_dir /path/to/KITTI/odometry/set/ --output_dir /path/to/output/directory/ --ckpt_file /path/to/pre-trained/model/file/**  , so where is **KITTI/odometry/set/**, you mean I should make own test set? how to make the test set ?
